### DISCUSSION TITLE
Download the Edgeglass mode design as image

### DISCUSSION THREAD

**Discussion by bishnupc** [toeverything/AFFiNE#8634] [General]
I see there are export options but it does export only the details in the page. If anything designed using the edgeglass mode, I dont see the same in the exported html file.
So is there any functionality to download as ong or svg for the designs and flows?

**Comment by fallsdevil** [comment]
exactly! I was wanting to download my whiteboard and I couldn't download in image format either, as it happens on tldraw and excalidraw, I asked a question about it recently, but I didn't get answers ... I'm just waiting for it to migrate for a for good to affine
